dojo.py

```diff
@@ -18,93 +19,85 @@ def create_room(self, room_type, room_names):
 		for name in room_names:
 			room_name = name.lower()
 			room_type = room_type.lower()
-			occupants = []
-			room_capacity = (6 if room_type=='office' else 4)
 
-			current_rooms = []
-			for room in self.rooms:
-				current_rooms.append(room['room_name'])
-			if room_name in current_rooms:
+			found = False
+			for room in self.all_rooms:
+				if room_name == room.room_name:
+					found = True
+			if found:
 				msg += "Room with name " + room_name + " already exists \n"
 			else:
 				if room_type in ('office', 'living'):
-					self.rooms.append({'room_name': room_name, 'occupants': occupants, 'type': room_type, 'room_capacity': room_capacity})
-					if room_type == 'office':
-						room = Office(room_name)
-						self.offices.append({'room_name': room_name, 'occupants': occupants, 'room_capacity': room_capacity})
-						msg += "Office " + room_name + " added successfully \n"
-					elif room_type == 'living':
-						room = LivingSpace(room_name)
-						self.livingSpaces.append({'room_name': room_name, 'occupants': occupants, 'room_capacity': room_capacity})
-						msg += "Living space " + room_name + " added successfully \n"
+					room = Room(room_name)
+					self.all_rooms.append(room)
+				if room_type == 'office':
+					room = Office(room_name)
+					self.offices.append(room)
+					msg += "Office " + room_name + " added successfully \n"
+				elif room_type == 'living':
+					room = LivingSpace(room_name)
+					self.livingSpaces.append(room)
+					msg += "Living space " + room_name + " added successfully\n"
 				else:
 					msg += "Wrong room type \n"
-		return msg
+			return msg
```

Comment:
Is the code working? The reason being, it seems to me that you're expecting a list. If a list has five items; what will happen to the four items once the first loop is completed and the function is terminated by a return command? 

I would also suggest that you make use of the print() function so that the user can see what's going on. I don't think a return statement will output something for the user